### PR TITLE
WT-10277 Unclear error when shutdown RTS fails

### PR DESCRIPTION
**Pull request by clarissecheah**
We don't check for a return value when calling RTS as part of shutdown. As such, currently if RTS fails during shutdown the logs only show a failure on connection close:

`
 2023-03-16T23:39:51.321+00:00 E WT 22435 [SignalHandler] "WiredTiger error message","attr":{"error":-700,"message":{"ts_sec":1679009991,"ts_usec":321424,"thread":"15378:0x7f2b35af6700","session_name":"WT_CONNECTION.close","category":"WT_VERB_DEFAULT","category_id":9,"verbose_level":"ERROR","verbose_level_id":-3,"msg":"__conn_close:1200:failure during close, disabling further writes","error_str":"error return: -700","error_code":-700}}
`

This can be non-obvious for someone trying to track down the cause of a close failure. I've a check for the return value and logging here in the case that RTS fails during shutdown. Now, this is logged in addition to the original connection closure logs:
 
`[j0:sec] | 2023-03-16T23:39:51.308+00:00 I WTRTS 22430 [SignalHandler] "WiredTiger message","attr":{"message":{"ts_sec":1679009991,"ts_usec":308615,"thread":"15378:0x7f2b35af6700","session_name":"WT_CONNECTION.close","category":"WT_VERB_RTS","category_id":31,"verbose_level":"DEBUG_1","verbose_level_id":1,"msg":"performing rollback to stable as part of shutdown failed with error code -700"}}`

### CODE REVIEW

**Comment by clarissecheah** [comment]
Another option that might be more suitable here actually would be to use `__wt_verbose_notice` instead. 

```

    WT_VERBOSE_NOTICE,      /*!< Messages for significant events in WiredTiger, usually worth
                            noting. */
```
@korteland, let me know if you have a preference.

**Comment by korteland** [comment]
Thanks! Yes, I think `WT_VERBOSE_NOTICE` would be more appropriate here. (While you're here, I think the comment is a bit redundant.)

File: src/txn/txn.c [hunk 1]
```diff
@@ -2422,6 +2422,11 @@ __wt_txn_global_shutdown(WT_SESSION_IMPL *session, const char **cfg)
               "[SHUTDOWN_INIT] performing shutdown rollback to stable, stable_timestamp=%s",
               __wt_timestamp_to_string(conn->txn_global.stable_timestamp, ts_string));
             WT_TRET(conn->rts->rollback_to_stable(session, cfg, true));
+
+            /* Log an error if rollback to stable fails here. */
+            if (ret != 0)
+                __wt_verbose(session, WT_VERB_RTS,
+                  "performing shutdown rollback to stable failed with code %d", ret);
```
Comment (korteland) [inline src/txn/txn.c:2429]: Would it be appropriate to use `__wt_strerror` here?

Comment (clarissecheah) [reply src/txn/txn.c]: I don't see why not! [1457a09](https://github.com/wiredtiger/wiredtiger/pull/8933/commits/1457a09053ba2de750c2016125d9401948051a2e)

File: src/txn/txn.c [hunk 2]
```diff
@@ -2422,6 +2422,12 @@ __wt_txn_global_shutdown(WT_SESSION_IMPL *session, const char **cfg)
               "[SHUTDOWN_INIT] performing shutdown rollback to stable, stable_timestamp=%s",
               __wt_timestamp_to_string(conn->txn_global.stable_timestamp, ts_string));
             WT_TRET(conn->rts->rollback_to_stable(session, cfg, true));
+
+            if (ret != 0)
+                __wt_verbose_notice(session, WT_VERB_RTS,
+                  WT_RTS_VERB_TAG_SHUTDOWN_RTS
+                  "performing shutdown rollback to stable failed with code %s",
```
Comment (korteland) [inline src/txn/txn.c:2429]: This won't quite match the regex because the equals sign is missing.

**Review by korteland — APPROVED** [review]
LGTM besides one thing, no need for a re-review.

File: src/txn/txn.c (hunk 2 above)
Comment (clarissecheah) [reply src/txn/txn.c:2429]: [59e896b](https://github.com/wiredtiger/wiredtiger/pull/8933/commits/59e896b0b7b2c47938d502926e61165bba974949)

**Pull request merged**